sdk/python/tests/unit/test_vector_store.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import pytest
+
+from tests.example_repos.example_feature_repo_1 import document_embeddings
+from tests.utils.rag_test_utils import MockVectorStore
+
+
+@pytest.fixture(autouse=True)
+def cleanup_milvus():
+    """Ensure Milvus resources are cleaned up between tests."""
+    yield
+    # This will run after each test
+    import gc
+
+    gc.collect()  # Force garbage collection to help release resources
+    # Add a small delay to ensure resources are released
+    import time
+
+    time.sleep(0.1)
+
+
+def test_vector_store_initialization(feature_store):
```

Comment:
feature_store fixture needs applied_permissions fixture as well, so it's failing in unit test. You might want to define your own fixture.

Reply:
Thanks for reviewing! Renamed this.